test/test_spark_torch.py

```diff
@@ -354,3 +354,20 @@ def test_torch_direct_parquet_train(self):
                     transformer = est.fit_on_parquet()
                     predictions = transformer.transform(df)
                     assert predictions.count() == df.count()
+
+    def test_make_list_if_not_list(self):
```

Comment:
If we move the function then this should be in `test_spark.py`.

Reply:
moved.